src/views/AddCourse.vue

```diff
@@ -15,6 +15,7 @@ const course = ref({
   description: ""
 });
 const message = ref("Enter data and click save");
+const errors = ref({});
 
 function addCourse() {
   CourseServices.addCourse(course.value)
```

Comment:
I would clear any existing errors before updating the error message(s):
![image](https://github.com/user-attachments/assets/2456732f-9d5a-400b-80c5-a9258a7a7d82)
![image](https://github.com/user-attachments/assets/80e2ad58-427f-4910-aa14-367402785f9c)
When I received an initial error, I fixed the error but "accidentally" messed up in another field. When hitting add again, I still had all the old errors, and the field in which the error happened was unclear at first glance

Reply:
fixed

Reply:
Looks good